ROADMAP.md

```diff
@@ -1,7 +1,45 @@
 # KF Serving 2019 Roadmap
-## Q2 2019
+## Q3 2019
+### v0.3 Performance and Stability (ETA: September 1, 2019)
+Objective: "Prevent feature regressions with 80% e2e test coverage against a live Cluster."
+* Automated End-to-End Tests
+    * Execute against a Kubeflow maintained GKE Cluster.
+    * Achieve >80% Test Coverage of Supported Features.
+
+Objective: "Prevent performance regressions across a known set of representative models."
+* Automated Performance Tests 
+    * Define a set of Models to test covering a wide array of usecases and frameworks.
```

Comment:
are we testing predictive accuracy or latency / CPU / memory load?
@gogowings fyi

Reply:
Infrastructure. These are on a known set of pretrained models.